out/build/x64-Debug/CMakeCache.txt

```diff
@@ -18,10 +18,10 @@
 CMAKE_BUILD_TYPE:UNINITIALIZED=Debug
 
 //No help, variable specified on the command line.
-CMAKE_CXX_COMPILER:FILEPATH=C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.23.28105/bin/HostX64/x64/cl.exe
+CMAKE_CXX_COMPILER:FILEPATH=D:/Logiciels/CMake/bin/cmake.exe
 
 //No help, variable specified on the command line.
-CMAKE_C_COMPILER:FILEPATH=C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.23.28105/bin/HostX64/x64/cl.exe
+CMAKE_C_COMPILER:FILEPATH=D:/Logiciels/CMake/bin/cmake.exe

```

Comment:
Do we have to commit these files? How about ignoring `out/`?

Reply:
Added to gitignore 👍